lib/minio/client.rb

```diff
@@ -10,23 +10,12 @@
 
 class Minio::Client
```

Comment:
Nice economy here, but may want to mention in the commit body text that it's strictly a refactor (right?) to get more economy in the code.